docs/multiform.rst

```diff
@@ -60,6 +60,13 @@ use an OrderedDict to define the :attr:`~MultiForm.form_classes`. ::
             ('profile', ProfileForm),
         ))
 
+An implementation of the attribute :attr:`~MultiForm.base_fields` is provided,
+but the keys are a combination of the subform keys and the field names rather
+than just the field names. The reason an implemention is provided at all, given
+it is merely an approximation, is interoperability with ``WizardView``, which
+introspects the values of :attr:`~MultiForm.base_fields`.  See
+:ref:`wizardview_guide`.
+
```

Comment:
This is almost perfect. I just think we should talk about what could possibly go wrong here.

Basically, the only thing that I know will go wrong is that if you are dynamically adding more form_classes to your MultiForm, it's possible that the WizardView will not realize that you have a FileField and will not set the `file_storage` attribute correctly.

I think we can just put a small warning here that says something along the lines of:

```rst
.. warning::		
		
    If you are having problems with the WizardView not realizing that your MultiForm includes
    FileFields, you may need to configure the ``file_storage`` attribute for your WizardView.
```

What do you think?